### ISSUE TITLE
Team must be manually activated via UI...

### ISSUE THREAD

**Issue by kepeter** [microsoftgraph/microsoft-graph-docs#9539]
Hi,

Using the sample code (C#), the created team must be activated manually via the UI (it displays a notification bar at the top, that states, that some members are waiting to join and there is an activation button next to it)...
If I do not do it, those members added via the API do not became really members and trying to use them will generate an error: for instance if I try to add one member to a private channel before activation it will fail with 'not member of the team' error...

![image](https://user-images.githubusercontent.com/695660/90332143-2d4f2600-dfc3-11ea-8632-628f31d467e9.png)

There is no such request if the team created via UI...

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 33803a53-4965-1c01-ea17-49e01b8378c5
* Version Independent ID: 9223513b-37d0-f9ea-f004-83f2222906b0
* Content: [Create team from group - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/team-put-teams?view=graph-rest-beta&tabs=http)
* Content Source: [api-reference/beta/api/team-put-teams.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/beta/api/team-put-teams.md)
* Product: **microsoft-teams**
* Technology: **microsoft-graph**
* GitHub Login: @nkramer
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by Nikitha-MSFT** [comment]
We cold not find the settings in teams UI for meeting in public ring. In which ring you are trying? Could you please conform?

**Comment by Nikitha-MSFT** [comment]
Are you trying to add members to the teams after immediate creation of team? Could you please conform?

**Comment by kepeter** [comment]
> Are you trying to add members to the teams after immediate creation of team? Could you please conform?

Yes. I do all the creation via API...
1. Create team
2. Add members/owners
3. Create standard channels
4. Create private channels
5. Add members to private channels

If I do not login to the UI and hit that blue 'Activate' button **5.** will fail with a message that the user I tray to add must be a members of the team...

**Comment by v-karapa** [comment]
Hi Kepeter,
Normally replication takes time to reflect. It takes 30mins to 24hrs depending on replication
is this happening for all teams ?
if it is happening for specific teams check owner having proper teams Teacher or Student license.

once the team is created. Every member and owner will receive email notification to join. atleast one owner should login to start the team, otherwise it will create automatically once replication happens.

**Comment by Nikitha-MSFT** [comment]
Class teams created from groups are in a state of "early class access", meaning the owners have access to the team but the members do not until the team is activated. Please find documentation regarding this [here](https://docs.microsoft.com/en-us/microsoftteams/remote-learning-edu#early-access-to-class-teams)

**Comment by kepeter** [comment]
@Nikitha-MSFT Thank you!
There is any way to activate it via API?

**Comment by kepeter** [comment]
@Nikitha-MSFT And almost forgot to ask the original question... Why I cannot add - prior activation - the members to private channels via API...

**Comment by kepeter** [comment]
It seems that this issue and https://github.com/microsoftgraph/microsoft-graph-docs/issues/9538 are bot boiled down to be the same...

**Comment by Nikitha-MSFT** [comment]
As described in the [documentation](https://docs.microsoft.com/en-us/microsoftteams/remote-learning-edu#how-do-i-activate-a-class-for-an-educator)
How do I activate a class for an educator?
`
Use the Update Team API and set the isMembershipLimitedToOwners property to false to activate the team on your educator’s behalf. Note that after a team is activated, it can't be reversed.`

**Comment by kepeter** [comment]
Which brings me this:
```
Error CS0117 'Team' does not contain a definition for 'IsMembershipLimitedToOwners'
```
Is seems like an API problem (as the documentation and the library are not on the same page)...

**Comment by kepeter** [comment]
This issue turned out to be a duplicate of https://github.com/microsoftgraph/microsoft-graph-docs/issues/9538
Closing here to simplify things...
A big thank to everyone tried to help...
